### ISSUE TITLE
How to disable Grpc Security

### ISSUE THREAD

**Issue by DoneSpeak** [LogNet/grpc-spring-boot-starter#206]
In my spring boot application, using spring security, I try to disable Grpc security with the following properties config, but fail.

```yml
grpc:
  server:
    security:
      enabled: false
```

and I use the following config to disable security
```java
@SpringBootApplication(exclude = GrpcServerSecurityAutoConfiguration.class)
```

I can't find any doc to tell me how to disable grpc security.

**Comment by jvmlet** [comment]
Meanwhile you can override the adapter with empty configure method...

**Comment by DoneSpeak** [comment]
> Meanwhile you can override the adapter with empty `configure` method...

Which class?

**Comment by jvmlet** [comment]
```java
@EnableGrpcSecurity
    public class GrpcSecurityConfiguration extends GrpcSecurityConfigurerAdapter {
        

        @Override
        public void configure(GrpcSecurity builder) throws Exception 
        }
    }
```

**Comment by jvmlet** [comment]
Set `grpc.security.auth.enabled` to `false`, will be released in 4.5.0

**Comment by jvmlet** [comment]
@DoneSpeak , `4.5.0` released
